tex/artikel/Gleichstellungsbuero.tex

```diff
@@ -1,35 +1,35 @@
+Zum Beispiel veranstalten wir regelmäßig Gender Breakfast Talks und Physikerinnen-Cafés. Dort können wir uns austauschen und unterstützen sowie über interessante und aktuelle Themen quatschen. Der Gender Breakfast Talk ist eine Veranstaltung explizit für alle Geschlechter – Gleichstellung ist nämlich kein Thema für Frauen allein. Während des Physikerinnen Cafés möchten wir jedoch eine für Frauen geschützte Umgebung schaffen und Hürden und Probleme besprechen, die speziell Frauen im Studium und in der Karriere begegnen.
 
-Alle Infos dafür findet ihr auf unserer Website \footnote{\url{www.uni-muenster.de/Physik/department/equality/}} oder bekommt ihr regelmäßig, indem ihr euch auf unseren entspannten Newsletter setzt. Dies geht ganz einfach über diesen QR-Code: \\
-\begin{center}
-    \includegraphics[width=\columnwidth]{res/gst_QR.png} %Newsletter QR-Code
-\end{center}
+Beim Gender Breakfast Talk starten wir immer mit einem wissenschaftlichen Input-Talk Gleichstellungmit neusten Ergebnissen aus der Gender \& MINT Forschung. Dazu laden wir eine Rednerin ein, die in diesem Gebiet forscht. So bekommt jede:r Zuhörende die Möglichkeit, die eigenen Gender-Kompetenzen auszubauen – keine Angst, Vorkenntnisse werden nicht benötigt. Wichtig ist uns dabei, dass der Talk wissenschaftlich ist. Das bedeutet, dass alle Aussagen auf Studien beruhen und fundiert sind. Dementsprechend wollen wir eine offene und vielseitige Diskussion führen, was auch bedeutet, dass wir uns kritisch hinterfragen. Danach sprechen wir über das Thema des Vortrags oder auch jedes andere Thema, das aufkommt, klären Fragen und teilen unsere Erfahrungen in bestimmten Situationen. Dazu gibt es auch noch ein kleines kostenloses Frühstück.
 
-Grundsätzlich gilt: Wir sind die ersten Ansprechpersonen, wenn es um Gleichstellungsthemen geht. Also ruft uns einfach an (251 83-33516), schreibt uns eine E-Mail (\textbf{\email{gleichstellung.physik@uni-muenster.de}}) oder kommt auf einen Kaffee rum (Raum: AP106) und wir helfen euch. \\
+Das Physikerinnen-Café findet immer statt, wenn im Allgemeinen Physikalischen Kolloquium eine Rednerin spricht. Am selben Tag ihres Vortrags treffen wir uns vorab und sprechen sowohl über ihren Karriereweg als auch über geschlechterbezogene Probleme, denen sie während ihres eigenen Werdegangs begegnet ist. Meistens haben diese schon weiter fortgeschrittenen Wissenschaftlerinnen super Tipps für uns Studentinnen.
 
+Auch für die Themen sexualisierte Diskriminierung, Belästigung und Gewalt sind wir zuständig. Dabei haben wir immer ein offenes Ohr für euch, gehen mit euren Sorgen sowie Nöten diskret um und kennen die nötigen Ansprechpersonen, wenn weitere Schritte eingeleitet werden sollen.
 
-\fibelsig{Anna Niemann und Janice Bode}
+\begin{multicols}{2}
+Alle Infos dafür findet ihr auf unserer \mbox{Website\footnotemark} oder bekommt ihr regelmäßig, indem ihr euch auf unseren entspannten Newsletter setzt. Dies geht ganz einfach über diesen QR-Code:
```

Comment:
,,mbox'' wird hier nicht übernommen - wollt ihr unbedingt etwas in einer Box, probiert mal \fbox{} oder - in fancy - \fcolorbox{}{}{} mit \fcolorbox{Rahmenfarbe}{Hintergrundfarbe (am besten natürlich 'white')}{Text}

Reply:
mbox an dieser Stelle schützt "Website" davor, - in ich meine es war "Websi-te" - silbengetrennt über zwei Zeilen aufzutauchen, ist also gut, dass die mbox unsichtbar ist

Reply:
Aber was für einen Sinn hätte 'mbox' dann, wenn nichts umrahmt wird?^^' 
Hab das bisher nie als eine Art ,,geschütztes Leerzeichen" gesehen

Reply:
mbox an dieser Stelle verhindert einen Zeilenumbruch im Wort "Webseite" mit einer komischen Silbentrennung von ich meine "Websi-te".